tests/queries/0_stateless/03461_pk_prefix_trivial_count.reference

```diff
@@ -0,0 +1,7 @@
+-- { echo ON }
+
+DROP TABLE IF EXISTS t;
+CREATE TABLE t(k String) ORDER BY k as select 'dst_'||number from numbers(1e6);
+SELECT count(*) FROM t WHERE k LIKE 'dst_kkkk_1111%';
+0
+DROP TABLE t;
```

Comment:
Can you please add a test for `%` as well?

`SELECT count(*) FROM t WHERE k LIKE 'dst%kkkk';`